src/App.js

```diff
@@ -36,7 +38,6 @@ export default function App() {
 
   const sortAlphabetically = storeItems => storeItems.sort(compare)
```

Comment:
You could refactor this by passing the event value to a function that calls your two different filter functions and then inside those functions set your state that is used to render the store.